custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -1079,15 +1079,19 @@ def get_mehsana_resistance_properties(column_mapping, row):
 
 
 def get_episode_regimen_change_history(column_mapping, row, episode_treatment_initiation_date):
-    put_on_treatment = column_mapping.get_value("date_put_on_mdr_treatment", row)
-    put_on_treatment = clean_date(put_on_treatment)
-    value = "{}: MDR/RR".format(episode_treatment_initiation_date)
-    if put_on_treatment:
-        value += "\n{}: {}".format(
-            put_on_treatment,
-            column_mapping.get_value("type_of_treatment_initiated", row)
-        )
-    return value
+    # TODO: This is odd Mehsana code
+    # put_on_treatment = column_mapping.get_value("date_put_on_mdr_treatment", row)
+    # put_on_treatment = clean_date(put_on_treatment)
+    # value = "{}: MDR/RR".format(episode_treatment_initiation_date)
+    # if put_on_treatment:
+    #    value += "\n{}: {}".format(
+    #        put_on_treatment,
+    #        column_mapping.get_value("type_of_treatment_initiated", row)
+    #    )
+    #return value
+    current_treatment_regimen = column_mapping.get_value("treatment_regimen", row)
+    if current_treatment_regimen:
+        return ("{}: " + current_treatment_regimen).format(episode_treatment_initiation_date)
```

Comment:
This will work, but is a bit confusing. A clearer way of writing it would have been:
```py
"{}: {}".format(episode_treatment_initiation_date, current_treatment_regimen)
```